src/client/storage.rs

```diff
@@ -0,0 +1,52 @@
+
+pub mod default_cache;
+
+/// A trait for giving a type a custom cache storage implementation
+///
+/// Sometimes, you want to use your own cache implementation to store records retrieved from the remote-settings server
```

Comment:
Since we provide a dumb storage implementation, we'll always want to have our own ;)